src/ploneintranet/search/browser/searchresults.py

```diff
@@ -22,7 +22,7 @@ def search_response(self):
             if form.get(facet):
                 facets[facet] = form.get(facet)
 
-        search_util = getUtility(ISiteSearch, name='example')
+        search_util = getUtility(ISiteSearch, name='zcatalog')
         response = search_util.query(
```

Comment:
Not that it should stop a merge of this, but as it stands this means we'd need to change view code if we change the backend. I thought we were attempting to avoid this.